python/dev_requirements.txt

```diff
@@ -0,0 +1,7 @@
+# Copyright (c) 2021, NVIDIA CORPORATION.
+
+clang==8.0.1
```

Comment:
In checking Pandas, they include everything needed for development (install / runtime dependencies) in this file, so maybe we should do the same?

Reply:
Sure, I can make similar change to cudf side too so that we remain consistent.

Reply:
Added it here 👍